Realm/Realm/DatabaseTypes/RealmObjectBase.cs

```diff
@@ -501,5 +506,99 @@ public Metadata(TableKey tableKey, IRealmObjectHelper helper, IDictionary<string
                 Schema = schema;
             }
         }
+
+        /// <summary>
+        /// A class that exposes a set of API to access the data in a managed RealmObject dynamically.
+        /// </summary>
+        public struct Dynamic
+        {
+            private readonly RealmObjectBase _realmObject;
+
+            internal Dynamic(RealmObjectBase ro)
+            {
+                _realmObject = ro;
+            }
+
+            public T Get<T>(string propertyName)
```

Comment:
I do get `Elements should be documented` warnings on this and all other `public` elements in this `struct`.

Reply:
Yes, it's not "ready" ready for review but wanted to put it up to discuss the API. Once we're aligned on that, I'll add API docs.

Reply:
👍